addons/hw_drivers/controllers/driver.py

```diff
@@ -358,6 +358,7 @@ def run(self):
             devices = updated_devices
             for path in [device_rm for device_rm in removed if device_rm in iot_devices]:
                 iot_devices[path].disconnect()
+                _logger.info('For device %s will be remove', path)
```

Comment:
i think the message isn't really helpful
maybe "Device %s is now disconnected"
and in the connection part "Device %s is now connected"